src/encryption/PrivateKeyDBFieldEncryption.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2018 ShipChain, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import EthCrypto from "eth-crypto";
+import { DBFieldEncryption } from "../entity/Wallet";
+
+const LOCAL_SECRET_KEY = process.env.LOCAL_SECRET_KEY || "0x0000000000000000000000000000000000000000000000000000000000000001";
```

Comment:
We probably should document what this does in `README.md` even though it has a default.